docs/content/concepts/identifier-use/_index.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Identifier Use
+description: Provides details on the scope an duniqueness of dientifiers used within the OSCAL models.
+suppresstopiclist: true
+weight: 50
+toc:
+  enabled: true
+sidenav:
+  focusrenderdepth: 2
+  activerenderdepth: 2
+  inactiverenderdepth: 2
+---
+
+This page reviews important concepts to be aware of when declaring or referencing identifiers in OSCAL models, with in-depth explanations of identifier uniqueness and scope.  
+
+### **Identifier Type** 
+By design, OSCAL supports [*machine-oriented*](/concepts/identifier-use/#machine-oriented) and [*human-oriented*](/concepts/identifier-use/#human-oriented) identifiers. The OSCAL models dictate which are used for different data items.
+
+#### Machine-Oriented
+
+*Machine-oriented* identifiers are implemented as Universally Unique Identifier (UUID) as defined by [RFC 4122](https://tools.ietf.org/html/rfc4122) and are commonly used to (programmatically) identify or refer to unique information within OSCAL models.  Because OSCAL’s *`machine-oriented`* identifiers utilize UUID data types, they can be issued without a central authority, and used across OSCAL instances with low risk of identifier collisions. 
```

Comment:
```suggestion
*Machine-oriented* identifiers provide a persistent identity for an entity within OSCAL the OSCAL models, which can be used in other locations within related OSCAL models to reference the associated entity. 

These identifiers are intended to be auto-generated by tools when the entity is initially created. In OSCAL, a machine-oriented identifier is implemented using a Universally Unique Identifier (UUID) as defined by [RFC 4122](https://tools.ietf.org/html/rfc4122). UUIDs were chosen because:

- Programming interfaces exist in most programming environments to generate a UUID
- UUIDs can be issued without a central authority
- UUIDs are represented in 128bits, providing for a large address space with low risk of identifier collisions for randomly generated values
```